src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -1,81 +1,178 @@
-  const gust = datasets.filter((dataset) => dataset.data_type.standard_name.includes("gust"))
-  const direction = datasets.filter((dataset) => dataset.data_type.standard_name.includes("direction"))
+/**
+ * Display wind datasets on a current conditions card
+ */
+export const DisplayWindCard: React.FunctionComponent<DisplayWindCardProps> = ({ platform, datasets, unit_system }) => {
+  const { speed, gust, direction } = pickWindDatasets(platform, datasets)
+  const { speed: speedTimeSeries, gust: gustTimeSeries } = pickWindTimeSeries(platform)
 
-  const speedAndGust = [...speed, ...gust]
-  if (speedAndGust.length < 1) {
+  if ((!speed && !gust) || !(speedTimeSeries || gustTimeSeries)) {
     return null
   }
 
-  const data_converter = converter(speedAndGust[0].data_type.standard_name)
+  const data_converter = converter(speedTimeSeries!.data_type.standard_name)
 
   let speedTitle: string = ""
-  if (speed.length > 0 && speed[0].readings.length > 0) {
+  if (speed) {
     speedTitle =
       " - " +
-      round(data_converter.convertToNumber(speed[0].readings[speed[0].readings.length - 1].reading, unit_system), 1) +
+      round(data_converter.convertToNumber(speed.timeSeries[speed.timeSeries.length - 1].reading, unit_system), 1) +
       " " +
       data_converter.displayName(unit_system)
   }
 
   let gustTitle: string = ""
-  if (gust.length > 0) {
+  if (gust) {
     gustTitle =
       " gusting to " +
-      round(data_converter.convertToNumber(gust[0].readings[gust[0].readings.length - 1].reading, unit_system), 1) +
+      round(data_converter.convertToNumber(gust.timeSeries[gust.timeSeries.length - 1].reading, unit_system), 1) +
       " " +
       data_converter.displayName(unit_system)
   }
 
   let directionTitle: string = ""
-  if (direction.length > 0 && direction[0].readings.length > 0) {
-    const reading = direction[0].readings[direction[0].readings.length - 1].reading
+  if (direction) {
+    const reading = direction.timeSeries[direction.timeSeries.length - 1].reading
     const compass = compassDirection(reading)
     directionTitle = " from " + compass[1]
   }
 
   return (
-    <Card>
-      <CardHeader>
-        Winds{speedTitle}
-        {gustTitle}
-        {directionTitle}
-      </CardHeader>
-      <CardBody style={{ padding: ".2rem" }}>
-        <WindTimeSeriesChart days={1} barbsPerDay={10} data={data} height={150} unit_system={unit_system} />
-      </CardBody>
-    </Card>
+    <Col {...cardProps}>
+      <Link to={observationLink(platform, "wind")}>
+        <Card>
+          <CardHeader>
+            Winds{speedTitle}
+            {gustTitle}
+            {directionTitle}
+          </CardHeader>
+          <CardBody>
+            <WindTimeSeriesChart
+              days={1}
+              barbsPerDay={10}
+              data={datasets}
+              height={150}
+              {...{ speed, gust, direction, unit_system }}
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)